.gitignore

```diff
@@ -14,6 +15,7 @@
 x64/
 build/
 bld/
+site/
```

Comment:
After doing https://github.com/reactiveui/ReactiveUI/pull/771/files#r29442457 you can remove this line.